db/db.py

```diff
@@ -207,14 +207,17 @@ def set_user_credit(user_uuid, credit_satoshis, deduct=False):
                 UPDATE users SET credit_satoshis = credit_satoshis - %(credit_satoshis)s WHERE user_uuid = %(user_uuid)s
             """
         else :
+            #I think this is the correct query: new credit_satoshis = old credit_satoshis + added credit_satoshis
             select_query = """
-                UPDATE users SET credit_satoshis = %(credit_satoshis)s WHERE user_uuid = %(user_uuid)s
+                UPDATE users SET credit_satoshis = credit_satoshis + %(credit_satoshis)s WHERE user_uuid = %(user_uuid)s
```

Comment:
Awesome! great catch. Let's clean up the comment a little bit:
`credit_satoshis = old credit_satoshis + added credit_satoshis` gets the point across concisely I think